tortoise/contrib/test/__init__.py

```diff
@@ -235,13 +231,24 @@ class TestCase(SimpleTestCase):
     separate transaction that will rollback on finish.
     """
 
+    # async def _setUpDB(self) -> None:
+    #     _restore_default()
+    #     self.transaction = await start_transaction()  # pylint: disable=W0201
+    #
+    # async def _tearDownDB(self) -> None:
+    #     _restore_default()
+    #     await self.transaction.rollback()
```

Comment:
If this is broken, then practically everything to do with transactions are broken…

Reply:
Oh wow, it creates a new connection, and does not honour the current connection.
This is with py3.7 and postgresql on my system.

Reply:
The context is not kept. Basically the same issue I had with my attempts.